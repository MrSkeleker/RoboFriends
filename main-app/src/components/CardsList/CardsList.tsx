import React from 'react';
import Card from '../Card/Card';

function CardsList(props: any) {
    const {robots} = props;
    return (
        <div className='pa3'>
            {robots?.map((robot: any) => (<Card key={robot.id} {...robot}/>))}
        </div>
    )
}

export default CardsList;