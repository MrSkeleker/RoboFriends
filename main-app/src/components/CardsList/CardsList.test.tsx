import {shallow} from 'enzyme';
import React from 'react';
import CardsList from './CardsList';

const testProps = {
    robots: [{
        id: -1,
        name: 'Test',
        email: 'test@test.com'
    }]
}

it('expect to render CardList Component', () => {
    expect(shallow(<CardsList {...testProps}/>)).toMatchSnapshot();
});