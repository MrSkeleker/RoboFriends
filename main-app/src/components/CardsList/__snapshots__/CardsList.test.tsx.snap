// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList Component 1`] = `
<div
  className="pa3"
>
  <Card
    email="test@test.com"
    id={-1}
    key="-1"
    name="Test"
  />
</div>
`;
